libraries/my-controls/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+
+  "compilerOptions": {
```

Comment:
>compilerOptions [](start = 3, length = 15)

could you move into the toolchain and use $extends?

Reply:
Good idea

---
In reply to: [216846178](https://github.com/Microsoft/rush-example/pull/1#discussion_r216846178) [](ancestors = 216846178)